pcap.go

```diff
@@ -361,7 +362,7 @@ func (p *Pcap) getNextPkt(pkt *Packet) {
 	pkt.Data = (*[C.MAX_PKT_CAPLEN]byte)(buf)[0:pkt.Caplen]
 	if int(binary.BigEndian.Uint16(pkt.Data[12:14])) == 0x0806 {
 		if pkt.Data[14+4] == 0 && len(pkt.Data) >= int(8+2*pkt.Data[14+4]+2*pkt.Data[14+5]) {
-			dlog.Infof("s:%d m:%d u:%d r:%d pktin [C:%d P:%d L:%d] Data %x", p.seq, p.max, p.used, pkt.Caplen, pkt.Partial, pkt.Len, bytes.Split(pkt.Data, []byte(",")))
```

Comment:
Just wondering, what did `r` represent?

Reply:
I don't know.  I removed that part of the string because otherwise it did not compile.